AcadeMeData/test_msgtags.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from AcadeMeData.models import MessageTags, Messages, User
+
+
+@pytest.fixture
+def user_example():
+    user_data = {'username': "username22123", 'password': "password", 'email': "user@example.com",
+                 'university': "RU",
+                 'degree': "CS"}
+    user = User.create_user(*user_data)
+    return user
+
+
+@pytest.fixture
+def generate_message(user_example, msgID=10, text='This is a test message yo'):
+    user = user_example
+    message = Messages(msgID=msgID, userID=user, text=text)
+    message.save()
+    return message
+
+
+@pytest.fixture
+def generate_msgtags(user_example, generate_message, id=1):
+    msg = generate_message
+    user = user_example
```

Comment:
see my comments above the fixture holds for you the return values

Reply:
FIXED